maths/prime_numbers.py

```diff
@@ -2,6 +2,33 @@
 import math
 
 
+def slow_primes(max: int) -> Generator[int, None, None]:
+    """
+    Return a list of all primes numbers up to max.
+    >>> list(primes(0))
+    []
+    >>> list(primes(-1))
+    []
+    >>> list(primes(-10))
+    []
+    >>> list(primes(25))
+    [2, 3, 5, 7, 11, 13, 17, 19, 23]
+    >>> list(primes(11))
+    [2, 3, 5, 7, 11]
+    >>> list(primes(33))
+    [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31]
+    >>> list(primes(10000))[-1]
```

Comment:
```suggestion
    >>> list(slow_primes(0))
    []
    >>> list(slow_primes(-1))
    []
    >>> list(slow_primes(-10))
    []
    >>> list(slow_primes(25))
    [2, 3, 5, 7, 11, 13, 17, 19, 23]
    >>> list(slow_primes(11))
    [2, 3, 5, 7, 11]
    >>> list(slow_primes(33))
    [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31]
    >>> list(slow_primes(10000))[-1]
```

Reply:
Sorry, I forgot to change these. It should be fixed now.